examples/pymc3_howto/lasso_block_update.ipynb

```diff
@@ -4,40 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        initvals = pm.find_MAP()
</pre><p>it might be best to get rid of this completely instead of updating to use initvals. Is there any reason to use the map as initial point?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I did, however as I side note, once I remove `initvals`, sometimes it runs correctly but also many times the sampling hangs and I get some warnings. Specifically, running:
```
with model:
    step1 = pm.Metropolis([beta1, beta2])
    step2 = pm.Slice([sigma, tau])
    idata = pm.sample(10000, step=[step1, step2])
```
I sometimes get three of these RuntimeWarnings
```
/home/user/miniconda3/envs/pymc-test-py39/lib/python3.9/site-packages/pymc/step_methods/metropolis.py:250: RuntimeWarning: overflow encountered in exp
  "accept": np.exp(accept),
```
and the sampling hangs.

Reply:
Does this happen during tuning or during sampling?

Reply:
<pre class="ql-syntax" spellcheck="false">`"accept": np.exp(accept)
</pre><p>`that's probably an extreme value in the acceptance probability during MCMC. Since you're seeing this go away with MAP initivals, my best guess would be that the prior is far away from the typical set.</p><p>Looking at the traceplot below, that's most probably somethin with the "tau" parameter. The default initval for this U(0,1) variable will be 0.5, but the posterior is <em>very</em> close to 0.</p>

Reply:
Thanks for the hints. The issue happens during sampling. Also, the lam scaling is equivalent to the original notebook, and I tried changing it to lam=1, leave only tau=HalfNormal(0,1) and some other combinations, but I still get this issue on most runs without initvals. You think I should put initvals back? Also, if I run pm.sample(10000) (without steps and initvals) everything works fine.